### PR TITLE
Add automapper

### PR DESCRIPTION
**Pull request by philpursglove**
Bringing in AutoMapper to do some of the model->viewmodel transformations

### CODE REVIEW

File: DDDEastAnglia/Controllers/SessionController.cs [hunk 1]
```diff
@@ -149,26 +152,21 @@ public virtual ActionResult DeleteConfirmed(int id)
         private SessionDisplayModel CreateDisplayModel(Session session, UserProfile profile)
         {
             var isUsersSession = Request.IsAuthenticated && session.SpeakerUserName == User.Identity.Name;
-            var tweetLink = CreateTweetLink(isUsersSession, session.Title, Url.Action("Details", "Session", new {id = session.SessionId}, Request.Url.Scheme));
+            var tweetLink = CreateTweetLink(isUsersSession, session.Title, Url.Action("Details", "Session", new { id = session.SessionId }, Request.Url.Scheme));
```
Comment (alastairs) [inline DDDEastAnglia/Controllers/SessionController.cs:155]: I wonder how much AutoMapper is buying us here: you've only saved a couple of lines of code with it.

Comment (philpursglove) [inline DDDEastAnglia/Controllers/SessionController.cs:155]: Yeah, I'll revert this out (or can I remove commit b73ae02 from the pull request?) - my thought process was along the lines of 'If I'm bringing in Automapper, I must justify bringing it in by using it as much as possible'

**Comment by alastairs** [comment]
I think this might be a bit premature in general, actually. It will certainly become more useful as we build out more functionality, but I think there's re-architecting to go in before this buys us enough to warrant its inclusion.

I'll hold off closing it without merge for the moment if you think there's a good reason to include it now.

**Comment by alastairs** [comment]
I've not heard any objections, so closing now.

**Pull request closed without merging**